### PR TITLE
Add command support (i3blocks compatible)

### PR DESCRIPTION
**Pull request by balkin**
Added command support.

### CODE REVIEW

**Comment by denbeigh2000** [comment]
Hi balkin,

Thanks for the pull request, this is really cool!

Please run `go fmt` over this first.
A meaningful title/description would also be nice, but I can see what this does from reading.

I have a couple of small issues with the change that I'll point out in-line as well.

File: packages/command/command.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	items[0].Name = g.Name
+	cmd := exec.Command(g.Command)
+	if (len(g.Instance) > 0) {
+		cmd.Env = []string{fmt.Sprintf("BLOCK_INSTANCE=%s", g.Instance)}
+	}
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		log.Panicf("Failed to execute %s: %v", g.Command, err)
```
Comment (denbeigh2000) [inline packages/command/command.go]: Don't call `log.Panicf` here - the bar will crash entirely. Use `log.Errorf` instead.

`return nil, err` here, and let the parent method handle the output.
Feel free to format the error differently to make it more user-friendly with `fmt.Errorf`

**Comment by balkin** [comment]
Thanks for your suggestions, I'm still new to Go :)
Fixed these issues

**Comment by denbeigh2000** [comment]
Thanks for your contribution! I did a tiny bit of refactoring and merged in your commits.

**Pull request closed without merging**